README.md

```diff
@@ -17,11 +17,15 @@ The plugin adds a global `fileLoader` DSL variable, which provides methods for l
 The `fileLoader` variable provides the following methods:
 * `fromGit(String libPath, String repository, String branch, String credentialsId, String labelExpression)` - loading of a single Groovy file from the specified Git repository
 * `withGit(String repository, String branch, String credentialsId, String labelExpression)` - wrapper closure for multiple files loading from a same Git repo
+* `fromSVN(String libPath, String repositoryUrl, String credentialsId, String labelExpression)` - loading of a single Groovy file from the specified GVN repository
+* `withSVN(String repositoryUrl, String credentialsId, String labelExpression)` - wrapper closure for multiple files loading from a same SVN repo
+
 * `load(String libPath)` - loading of an object from a Groovy file specified by the relative path. Also can be used within `withGit()` closure to load multiple objects at once
 
 Parameters:
 * `libPath` - a relative path to the file, ".groovy" extension will be added automatically
 * `repository` - string representation of a path to Git repository. Supports all formats supported by [Git Plugin](https://wiki.jenkins-ci.org/display/JENKINS/Git+Plugin)
+* `repositoryUrl` - string representation of a path to SVN repository.
```

Comment:
Maybe better to use the same parameter as Git, but to rework the documentation a bit (E.g. add subbullets for each SCM type)